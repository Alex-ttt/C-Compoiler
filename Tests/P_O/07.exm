while () ..
Condition:
    a
  &
    1
Body:
  Statements:
      *
        b
        [
          0
        ]
    =
      a
  Symbols Table:
          b array[20] of pointer to int
