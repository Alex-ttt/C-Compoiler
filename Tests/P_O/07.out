 Symbols Table:
   a int
   main  int function main(
  )
    {
      Statements:
      while () ..
      Condition:
          a
        &
          1
      Body:
        {
          Statements:
            *
              b
              [
                0
              ]
          =
            a
          Symbols Table:
                      b array[20] of pointer to int
        }
    }
